src/_locales/zh_CN/messages.json

```diff
@@ -439,6 +439,25 @@
     "hash": "2aff10770796694827ec4253860e0771"
   },
 
+  "translatorCredit": {
+    "message": "该扩展简体中文翻译由 Cye3s 完成",
+    "description": "The credit text for the translator. See https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translator-credit-inside-of-add-on for how to translate this.",
+    "placeholders": {
+      "translators": {
+        "content": "$1",
+        "example": "<a href=\"https://github.com/Cye3s/\">@Cye3s</a>"
+      }
+    }
+  },
+  "translatorLink": {
+    "message": "https://github.com/Cye3s",
+    "description": "The link to the translator's GitHub profile."
+  },
+  "translatorUsername": {
+    "message": "Cye3s",
+    "description": "The username that the translator wants to be referred to."
+  },
+
```

Comment:
Apropos missing translations, I think `contributorsThanks` (and similar) are also missing here.

Also remember you can use [the online translator tool](https://lusito.github.io/web-ext-translator/?gh=https://github.com/Cye3s/offline-qr-code), even if you just want to see which translations are missing.